CIP-0035/README.md

```diff
@@ -0,0 +1,302 @@
+---
+CIP: 35
+Title: Plutus Core Evolution
+Authors: Michael Peyton Jones <michael.peyton-jones@iohk.io>
+Comments-Summary: No comments
+Comments-URI: 
+Status: Draft
+Type: Process
+Created: 2022-02-09
+License: CC-BY-4.0
+---
+
+# Evolution of Plutus Core in the Cardano ledger
+
+## Abstract
+
+This CIP proposes a process for proposing changes to Plutus Core itself, and the parts of the Cardano ledger which relate to it.
```

Comment:
What specifically are these? Considering Plutus' nontrivial dependency footprint, it would be good to have a list of these, even if it's not 100% exhaustive.

Reply:
Does the list in "Types of change" not answer this question?

Reply:
I guess it's this sentence that's mostly confusing. I thought "This document considers the following types of change: <explicit list of changes>" was pretty explicit but okay.

Reply:
i think the direct question here is "My change requires modifications to `cardano-base` as well as the plutus interpreter, does this require two separate CIP procedures?"

Reply:
this seems to be answered 'No' in the below longform comment

Reply:
What I was trying to clarify is what in the text makes it seem that you *would* need a CIP to modify `cardano-base`, given that "changes to dependencies" is not listed as a type of change covered by the CIP.